spec/dummy/config/initializers/application_controller_renderer.rb

```diff
@@ -0,0 +1,6 @@
+# Be sure to restart your server when you modify this file.
+
+# ApplicationController.renderer.defaults.merge!(
+#   http_host: 'example.org',
+#   https: false
+# )
```

Comment:
I'd be fine with leaving out all comment only files that aren't directly necessary for the dummy app in order to reduce the noise and just have the very essential needed to make this work. I've done it that way in another gem of mine: https://github.com/fphilipe/warden-github-rails/tree/master/spec/rails_app